script/miRNAture

```diff
@@ -71,29 +71,38 @@ GetOptions (
 pod2usage ( -verbose => 1 ) if $help;
 pod2usage ( -verbose => 2 ) if $man;
 pod2usage ( -verbose => 99, -sections => "VERSION" ) if $ver;
-pod2usage ( -message => "Seems that you did pass an empty flag, please refer to documentation.", -verbose => 1) if (keys %opts == 0 && length $specie_genome_path == 0);
+pod2usage ( -message => "Seems that you did pass an empty flag, please refer to documentation.", -verbose => 1) if (keys %opts == 0 && length $species_genome_path == 0);
 
 @strategy = split (/,/, join(',',@strategy));
 
 # Relative paths correction for input paths
-if ($subset_models && $subset_models =~ /\/\.\.\/|^\.\//){
+if ($subset_models && $subset_models !~ /^\/\.+\//){
 	$subset_models = abs_path($subset_models);
 }
-if ($path_blast_queries =~ /\/\.\.\//){
+if ($user_folder && $user_folder !~ /^\/\.+\//){
+	$user_folder = abs_path($user_folder);
+}
+if ($path_blast_queries !~ /^\/\.+\//){
 	$path_blast_queries = abs_path($path_blast_queries);
 }
-if ($work_folder =~ /\/\.\.\//){
+if ($work_folder !~ /^\/\.+\//){
 	$work_folder = abs_path($work_folder);
 }
-if ($specie_genome_path =~ /\/\.\.\//){
-	$specie_genome_path = abs_path($specie_genome_path);
+if ($species_genome_path !~ /^\/\.+\//){
+	$species_genome_path = abs_path($species_genome_path);
+}
+if ($mirfix_path && $mirfix_path !~ /^\/\.+\//){
+	$mirfix_path = abs_path($mirfix_path);
+}
+if ($data_path !~ /^\/\.+\//){
+	$data_path = abs_path($data_path);
 }
 
 my $config_file = Mir::ConfigurationFile->new (
 	stage => $stagesRun,
-	specie_name => $name_specie,
-	specie_tag => $specie_tag,
-	specie_genome => $specie_genome_path,
+	specie_name => $name_species,
```

Comment:
Would make sense to replace all specie_ with species_
also in other modules for consistency

Reply:
Changed on flags and modules.